src/Fabulous/Builders.fs

```diff
@@ -125,36 +125,45 @@ type Content<'msg> = { Widgets: MutStackArray1.T<Widget> }
 type CollectionBuilder<'msg, 'marker, 'itemMarker> =
     struct
         val WidgetKey: WidgetKey
-        val Scalars: StackList<ScalarAttribute>
         val Attr: WidgetCollectionAttributeDefinition
+        val Scalars: AttributesBundle
 
-        new(widgetKey: WidgetKey, scalars: StackList<ScalarAttribute>, attr: WidgetCollectionAttributeDefinition) =
+        new(widgetKey: WidgetKey, attr: WidgetCollectionAttributeDefinition, scalars: AttributesBundle) =
             { WidgetKey = widgetKey
               Scalars = scalars
               Attr = attr }
```

Comment:
Good move, it is best to keep the same ordering of the parameters. Just watch out for the correct naming.
Scalars != Attributes

Also, it would be best to keep the original constructor as a short-hand for existing code.

```suggestion
        new(widgetKey: WidgetKey, attr: WidgetCollectionAttributeDefinition, attributes: AttributesBundle) =
            { WidgetKey = widgetKey
              Attr = attr
              Attributes = attributes }
              
        new(widgetKey: WidgetKey, attr: WidgetCollectionAttributeDefinition, scalars: StackList<ScalarAttribute>) =
            { WidgetKey = widgetKey
              Attr = attr
              Attributes = AttributesBundle(scalars, ValueNone, ValueNone) }
```

Reply:
Technically, we could release this change to the current v2 release, but we would need to keep the parameters ordering

Reply:
Yes would be good to release in 2.x also